tests/page_test.go

```diff
@@ -679,6 +679,15 @@ func TestPageURL(t *testing.T) {
 	assert.Regexp(t, "http://.*/get", p.URL())
 }
 
+func TestPageClose(t *testing.T) {
```

Comment:
Should we also add a test for pages created via a browser context?

Reply:
Resolved in afbb6991d1cd837e62d64d04d7bc3f92e24aeec5